src/data/list/chain.lean

```diff
@@ -72,6 +72,29 @@ theorem chain_map_of_chain {S : β → β → Prop} (f : α → β)
   (p : chain R a l) : chain S (f a) (map f l) :=
 (chain_map f).2 $ p.imp H
 
+theorem chain_pmap_of_chain {S : β → β → Prop} {p : α → Prop}
```

Comment:
`chain.pmap`?

Reply:
Is there a reasonable `iff` here?

Reply:
> `chain.pmap`?

Sure, shall I change the existing `chain_map_of_chain` as well then?

> Is there a reasonable iff here?

I couldn't figure out how to make one, if you have a suggestion I don't mind changing.